core/lib/basic_types/src/lib.rs

```diff
@@ -122,7 +122,7 @@ impl TryFrom<U256> for AccountTreeId {
 
 /// ChainId in the ZKsync network.
 #[derive(Copy, Clone, Debug, Serialize, PartialEq, Eq, PartialOrd, Ord, Hash)]
-pub struct L2ChainId(u64);
+pub struct L2ChainId(pub u64);
```

Comment:
What's the cause of this change? The wrapped value was intentionally made private because not all `u64` represent a valid L2 chain ID; see `L2ChainId::new`.